pkgs/games/steam/fhsenv.nix

```diff
@@ -291,6 +291,7 @@ in buildFHSEnv rec {
       description = "Steam dependencies (dummy package, do not use)";
     };
 
+  passthru.steamargs = args;
```

Comment:
This change is not compatible. Either annotate the args up to with `@args` just drop it; it's unrelated.

Reply:
Removed the commit in question.